src/client/aureliaCLICommands.ts

```diff
@@ -12,6 +12,7 @@ export default class AureliaCliCommands {
       vscode.commands.registerCommand('extension.auGenerate', () => this.auGenerate(outputChannel)),
       vscode.commands.registerCommand('extension.auTest', () => this.runCommand(['test'], outputChannel, false)),
       vscode.commands.registerCommand('extension.auBuild', () => this.auBuild(outputChannel)));
```

Comment:
Oops, my bad. I didn't see that I had added a trailing comma after the statement was completed; and sadly I didn't run the build after making the changes. I did run the tests however, but of course no tests hit this file. This is my first pull request anywhere. Hopefully my next one will be better. :-)